frontend/vuejs/src/views/Subjects/ListSubjects.vue

```diff
@@ -3,146 +3,179 @@
     <div>
       <projects-table></projects-table>
     </div>
+    <div class="container-fluid mt-2">
+      <div class="row">
+        <div class="col">
+          <div class="card shadow">
+            <div class="card-header border-0">
+              <div class="row align-items-center">
+                <div class="col">
+                  <h3 class="mb-0">
+                    {{ $t("list_subjects.title.list_subjects") }}
+                  </h3>
+                  <router-link
+                    id="add"
+                    class="btn btn-success"
+                    v-bind:to="'/addSubjects/'"
+                    >{{
+                      $t("list_subjects.title.add_new_subject")
+                    }}</router-link
+                  >
```

Comment:
![image](https://user-images.githubusercontent.com/67667833/99748580-61c90580-2b0f-11eb-9a1a-e8458d70a141.png)
v-bind:to => :to
dùng router name thay cho router path